recipes/diffpy.fourigui/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "diffpy.fourigui" %}
+{% set version = "0.1.0rc2" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Fixed - `conda_build_config.yaml` created for `rc` release.